disputer/disputer.js

```diff
@@ -71,37 +73,47 @@ class Disputer {
           at: "Disputer",
           message: "Cannot dispute liquidation: not enough collateral (or large enough approval) to initiate dispute.",
           id: disputeableLiquidation.id,
-          sponsor: disputeableLiquidation.sponsor
+          sponsor: disputeableLiquidation.sponsor,
+          error: error
         });
+        continue;
+      }
 
-        Logger.debug({
+      try {
```

Comment:
If an error throws on the send, I just don't add it to the promises array

Reply:
Is it possible for it to error on send? Don't all (or most) async errors get thrown to the .catch() method on a the promise? And if so, won't they miss this try catch and end up being uncaught on the `await Promise.all` line?

Reply:
you're right, I added a `.catch` error handler to the `.send`, so that the Promise.all won't throw and will log the error instead